cypress/e2e/udemy_automation-test-store/contact-us.js

```diff
@@ -1,5 +1,7 @@
 /// <reference types='cypress' />
 
+// not important change, just to force github to show this file in diff.
+
 describe('Test Contact Us for via Automation Test Store', () => {
     it('Submit a successful submission via contact us form and go back to front page', () => {
```

Comment:
FYI: it is customary to write a title of a case starting with "should" so it would kinda could be read like:

"it should submit a successful submission..."